adapters/bmtm/params_test.go

```diff
@@ -2,33 +2,35 @@ package bmtm
 
 import (
 	"encoding/json"
+	"fmt"
 	"testing"
 
 	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestValidParams(t *testing.T) {
 	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
 	if err != nil {
-		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+		assert.NoError(t, err, fmt.Sprintf("Failed to fetch the json-schemas: %s", err.Error()))
 	}
 
 	for _, validParam := range validParams {
 		if err := validator.Validate(openrtb_ext.BidderBmtm, json.RawMessage(validParam)); err != nil {
```

Comment:
Please remove the `err != nil` check as suggested above. It is no longer needed with `assert.NoError`. You can just have the following:
```
err := validator.Validate(openrtb_ext.BidderBmtm, json.RawMessage(validParam))
assert.NoError(t, err, fmt.Sprintf("Schema rejected brightMountainMedia params: %s", validParam))
```